lib/routes.js

```diff
@@ -88,6 +88,13 @@ const setupRoutes = function setupRoutes() {
       socket.progress = (data) => {
         send(prep({ action: 'progress', data }));
       };
+      
+      socket.reload = (data) => {
+        if (options.hmr || options.liveReload) {
```

Comment:
Also, we could move all this if statement (which is a duplicate that appears on the done event), into a function and reuse on both places

Reply:
I made this change. Thanks for the suggestion.